### ISSUE TITLE
Implement map view

### ISSUE THREAD

**Issue by cafootitt** [OpenSRP/opensrp-client-eusm#3]
Mockup: https://marvelapp.com/prototype/7d6gbea/screen/73387483

- [x] Menu button opens side menu
- [x] Search bar allows user to search by service point name
- [x] LIST button takes the user to the List view
- [x] Service points with GPS coordinates are displayed on the map in the district that the user has selected in the district dropdown menu in the side menu
- [x] Service points are represented by their icon (separate issue to be created)
- [x] The color of the service point corresponds to the item task status (green = all tasks complete; yellow = tasks in progress; and grey = no tasks done yet)
- [x] Service point icon changes to blue when the point is selected
- [x] When a service point is selected, a dialog opens showing the service point details:
-- Service point icon
-- Name
-- Type
-- Distance in kilometers the service point's GPS coordinates are away from the user's current location
-- Commune
-- Number of tasks in the task list
- [x] View inventory button takes the user to the service point inventory page
- [x] Handle map life-cycle between different actions/movements

**Comment by cafootitt** [comment]
16 Dec 2020 build:

Looking really great, just some small points:

- [ ] Search bar should have a cancel option
- [ ] I forgot to include this in the implementation issue (apologies!) but can we make the service point icon in the dialog the same color as the service point on the map (i.e. grey, yellow, or green, depending on task status?)
- [ ] I forgot to include this in the implementation issue (apologies!) but can we make the task status in the dialog the same color as the service point on the map (i.e. grey, yellow, or green, depending on task status?). So "In progress" would be yellow, while "Completed" would be green.

**Comment by bennsimon** [comment]
@cafootitt noted

**Comment by cafootitt** [comment]
Feedback from Matt:

Make the service point icons dynamically size to the zoom level of the map view.

**Comment by bennsimon** [comment]
@cafootitt Created this issue to track map icons issue https://github.com/OpenSRP/opensrp-client-eusm/issues/41.

**Comment by cafootitt** [comment]
Thanks, Benn.
